tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -0,0 +1,250 @@
+            setTimeForEachTask();
+        }
+        for (TimeStatisticForOneTask time: timesForEachTask) {
+            Long timeTakenInMilliseconds = time.getTimeTakenInMilliseconds();
+            Long milliseconds = timeTakenInMilliseconds % MILLISECONDS_IN_SECOND;
+            Long seconds = timeTakenInMilliseconds / MILLISECONDS_IN_SECOND;
+            Long minutes = seconds / SECONDS_IN_MINUTE;
+            seconds = seconds % SECONDS_IN_MINUTE;
+
+            out("");
+            out("TASK NAME: " + time.getTaskName());
+            out("START TIME: " + time.getStartTime().toString());
+            out("END TIME: " + time.getEndTime().toString());
+            out("DURATION: " + minutes + " minutes " + seconds + " seconds " + milliseconds + " milliseconds");
+        }
+        out("");
+        Long totalTimeTakenMilliseconds = getTotalTimeInMilliseconds();
+        Long milliseconds = totalTimeTakenMilliseconds % MILLISECONDS_IN_SECOND;
+        Long seconds = totalTimeTakenMilliseconds / MILLISECONDS_IN_SECOND;
+        Long minutes = seconds / SECONDS_IN_MINUTE;
+        seconds = seconds % SECONDS_IN_MINUTE;
+        out("TOTAL TIME: " + minutes + " minutes " + seconds + " seconds " + milliseconds + " milliseconds");
+    }
+
+    public void printRunStatistics() {
+        if (isWorkflowFinished()) {
+            out("RUN STATISTICS:");
+            out("ENTRY NAME: " + getCompleteEntryName());
+            out("END STATE: " + state);
+            if (state.equals("COMPLETE")) {
+                printTimeStatistic();
+            }
+        } else {
+            errorMessage("Workflow is not finished, statistic are not available yet", COMMAND_ERROR);
+        }
+    }
+
+    private String getTotalTimeInISO861Standard() {
+        if (timesForEachTask == null) {
+            setTimeForEachTask();
+        }
+        Long totalTimeMilliseconds = getTotalTimeInMilliseconds();
+        Duration duration = Duration.ofMillis(totalTimeMilliseconds);
+        return duration.toString();
+    }
+
+    private Execution.ExecutionStatusEnum getExecutionStatus() {
+        if ("COMPLETE".equals(state)) {
+            return Execution.ExecutionStatusEnum.SUCCESSFUL;
+        } else {
+            return Execution.ExecutionStatusEnum.FAILED_RUNTIME_INVALID;
+        }
+    }
+
+    public void uploadRunInfo() {
+        List<Execution> executions = new ArrayList<>();
+        Execution execution = new Execution();
+        execution.setExecutionStatus(getExecutionStatus());
+
+        // Only include the time metric if the workflow successfully ran
+        if (getExecutionStatus() == Execution.ExecutionStatusEnum.SUCCESSFUL) {
+            execution.setExecutionTime(getTotalTimeInISO861Standard());
+        }
+
+        executions.add(execution);
+
+        extendedGa4GhApi.executionMetricsPost(executions, Partner.OTHER.name(), getEntryNameForApi(), version, "this was generated with tooltester [TEST]");
```

Comment:
There's a `Partner.AGC` that you can use since these workflows were executed on AGC

Reply:
Corrected. Thanks for spotting this.